pcdsdevices/pim.py

```diff
@@ -372,32 +372,43 @@ class PPMPowerMeter(BaseInterface, Device):
     calibrated for each power meter in units of volts per watt
     is used to calculate the actual energy in the following way:
 
-    `calibrated_mj` = 1000 * (Signal - Background) / (Responsivity * Beam_Rate)
-    """
+    `calibrated_mj` = (Signal - Background) / (Responsivity * Beam_Rate)
+    """
+    def __init__(self, prefix, *, name, **kwargs):
+        import sympy.physics.units as units
+        from sympy.physics.units.prefixes import micro, milli
+        from sympy.physics.units.quantities import Quantity
+        millijoule = mJ = Quantity("millijoule", abbrev='mJ')
+        millijoule.set_global_relative_scale_factor(milli, units.joule)
+        microjoule = uJ = Quantity("microjoule", abbrev='uJ')
+        microjoule.set_global_relative_scale_factor(micro, units.joule)
+        setattr(units, 'mJ', mJ)
```

Comment:
So what we're doing here is manually adding a custom unit to the `sympy.physics.units` module?  This is almost certainly now how sympy is meant to be used, and import order can make this inconsistent.

The definition of these should probably go into `pcdsdevices.utils`, with maybe a modification of how we acquire units (First look for native sympy units, then look for our custom defined units)